Refit/RestService.cs

```diff
@@ -101,9 +105,9 @@ public T GetContentAs<T>()
                 default(T);
         }
 
-        public static async Task<ApiException> Create(HttpResponseMessage response, RefitSettings refitSettings = null)
+        public static async Task<ApiException> Create(string url, HttpMethod httpMethod, HttpResponseMessage response, RefitSettings refitSettings = null)
```

Comment:
Change the signature here to take an `HttpRequest` instead of `url` and `httpMethod`, then any future additions from the request won't have to touch the calling code.